### ISSUE TITLE
Add onion skin global toggle behavior

### ISSUE THREAD

**Issue by Jose-Moreno** [pencil2d/pencil#1085]
### Issue Summary

This is something more of a user experience issue that has been requested here and there. Onion skinning in most programs is normally turned on / off reminiscent of how a light table would. After that you are to handle the inclusion / exclusion of either previous or next frames relative to your current frame.
Consider this I'd like to ask to have a single switch (and shortcut) for toggling the onion skin globally, and leave the current PREV and NEXT functionality to be perceived as filters. This way there would be the following Onion Skin states: All OFF, All ON, PREV ONly, NEXT ONly

So basically we would end up having:

- [ ] A global switch to turn onion skin ON / OFF (w/ shortcut)
- [x] The PREV onion skin switch (w/shortcut)
- [x] The NEXT onion skin switch (w/shortcut)

**Comment by scribblemaniac** [comment]
I think there is potential for confusion if we have a global switch and a prev/next switch. If a user turns on next onion switching, they expect to see onion skins of the next frame. Image a scenario where the global switch is off, and one of the local switches is on but doing nothing. So it can be confusing, especially if they don't realize there is a global switch.

One solution would be to allow both the next and previous onion skin switches to be mapped to the same shortcut. 

An alternative acceptable solution I think would be to have an all on/all off action which simply changes the prev and next switch, but doesn't have any state of its own. The problem with this is, what if there is one on and one off? Should it turn them both on or both off. An action that just toggles both switches is possible too.

**Comment by CandyFace** [comment]
@scribblemaniac To me the natural effect of enabling or disabling the global onion skin would render the previous and next onion skin button gray aka. disabled too.
